### PR TITLE
Refactor/gubled

### PR DESCRIPTION
**Pull request by Bogh**

### CODE REVIEW

File: gubled/gubled.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ type Args struct {
 	GcmApiKey   string `arg:"--gcm-api-key: The Google API Key for Google Cloud Messaging" env:"GUBLE_GCM_API_KEY"`
 }
 
-var ValidateStoragePath = func(args Args) error {
+func ValidateStoragePath(args Args) error {
```
Comment (smancke) [inline gubled/gubled.go]: Having this creator/factory-functions as global, public variables was intended:
This gives a user of guble the possibility, to write its own main package, 
overwrite one or two of this functions and start guble with customized implementations of those modules.

Comment (Bogh) [inline gubled/gubled.go]: Cool. Makes sense now :D. I'll revert this.

File: gubled/gubled.go [hunk 2]
```diff
@@ -111,10 +111,7 @@ func Main() {
 	service := StartupService(args)
 
 	waitForTermination(func() {
-		err := service.Stop()
-		if err != nil {
-			guble.Err("Service: ", err)
-		}
+		service.Stop()
```
Comment (smancke) [inline gubled/gubled.go]: This change was useful .. should not be reverted ;-)

Comment (Bogh) [inline gubled/gubled.go]: Done

**Pull request merged**